packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/VirtualAssistant/VirtualAssistantConversationAlert.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import VirtualAssistant from '@patternfly/virtual-assistant/dist/dynamic/VirtualAssistant';
+import ConversationAlert from '@patternfly/virtual-assistant/dist/esm/ConversationAlert'
+
+export const BasicExample: React.FunctionComponent = () => (
+  <VirtualAssistant >
+    <ConversationAlert />
```

Comment:
Please add a sample title to have something in the preview

![image](https://github.com/patternfly/virtual-assistant/assets/3845764/b781e197-2856-4053-9467-469e6a501d00)